Xenon/Xe_Main.cpp

```diff
@@ -98,6 +103,7 @@ int main(int argc, char* argv[])
 	hdd.Initialize("HDD", HDD_DEV_SIZE);
 	smc.Initialize("SMC", SMC_DEV_SIZE);
 
+	/***************Add PCi devices***************/

```

Comment:
And PCi should be PCI, as those are the initials for Preipheral Interconnect Interfce.